src/Symfony/Bundle/WebServerBundle/Command/ServerStatusCommand.php

```diff
@@ -31,12 +32,11 @@ class ServerStatusCommand extends ServerCommand
     protected function configure()
     {
         $this
+            ->setName('server:status')
             ->setDefinition(array(
-                new InputArgument('address', InputArgument::OPTIONAL, 'Address:port', '127.0.0.1:8000'),
-                new InputOption('port', 'p', InputOption::VALUE_REQUIRED, 'Address port number', '8000'),
```

Comment:
Introducing a new composer package with a deprecation by default, is probably a bad idea? It should be done from the framework bundle.